crm/modules/payment/payment.inc.php

```diff
@@ -350,7 +350,7 @@ function payment_data ($opts = array()) {
                     $clause .= "`created` ";
                     break;
                 default:
-                    continue;
+                    continue 2;
```

Comment:
As I understand it, it forces the continuation of the loop - https://www.php.net/manual/en/control-structures.continue.php

Reply:
Yes, continue at this point in the code will act as a break, you need continue 2 to force the next iteration of the loop.